### PR TITLE
Depend on elm test

### PR DESCRIPTION
**Pull request by stoeffel**
#51

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -6,9 +6,19 @@ process.title = processTitle;
...
 
 // stateful things
 var cliModel = init(argv);
 
-cliModel.run(cliModel);
+cliModel.run(cliModel, function() {
+  var exit = childProcess.spawnSync(path.join(__dirname, '../node_modules/.bin/elm-test'),
+    {
+      cwd: process.cwd(),
+      stdio: 'inherit'
```
Comment (stoeffel) [inline bin/cli.js]: no colors from elm-test 😢

**Comment by stoeffel** [comment]
I've one more to review:-) sorry and thanks Noah! also feel free to unassign yourself.

**Comment by stoeffel** [comment]
reminder for myself: squash this pr

File: bin/runner.js [hunk 2]
```diff
@@ -101,6 +102,10 @@ function generate(model, allTestsGenerated) {
   });
 }
 
+function cleanup(model) {
+  rimraf.sync(model.testsDocPath);
```
Comment (eeue56) [inline bin/runner.js]: I worry about doing this freely: what if '/home/noah/' ends up as the testsDocPath?

Comment (stoeffel) [reply bin/runner.js]: totally and `Doc` is not that special.

Comment (stoeffel) [reply bin/runner.js]: I've renamed the folder `Doc` to `VerifyExamples`. This partially addresses your concern and is also nice in error messages.

fish___users_stoeffel_elm-verify-examples

Comment (eeue56) [reply bin/runner.js]: 👍

File: bin/cli.js [hunk 3]
```diff
@@ -6,9 +6,19 @@ process.title = processTitle;
...
+var childProcess = require('child_process');
+var path = require('path');
 
 
 // stateful things
 var cliModel = init(argv);
 
-cliModel.run(cliModel);
+cliModel.run(cliModel, function() {
+  var exit = childProcess.spawnSync(path.join(__dirname, '../node_modules/.bin/elm-test'),
```
Comment (eeue56) [inline bin/cli.js]: Why not default to global elm-test? In CI, elm-test is often installed globally

Comment (eeue56) [reply bin/cli.js]: Or rather: fallback to global, prefer local

Comment (stoeffel) [reply bin/cli.js]: sgtm!

**Pull request merged**